src/openMVG/dataio/AlembicImporter.cpp

```diff
@@ -170,6 +170,7 @@ bool AlembicImporter::readCamera(IObject iObj, M44d mat, sfm::SfM_Data &sfmdata,
   // Check if we have an associated image plane
   ICompoundProperty userProps = getAbcUserProperties(cs);
   std::string imagePath;
+  std::vector<int> sensorSize_pix;
```

Comment:
std::vector<unsigned int> sensorSize_pix = {2048, 2048};
And remove `sensorWidth_pix`.